.travis.yml

```diff
@@ -1,21 +1,44 @@
 sudo: required
 dist: trusty
 
+language: ruby
+jobs:
+  include:
+    script:
+      - set -e
```

Comment:
set -e is used for shell scripts to exit with non zero value at once if any of the lines in the script gives a non zero value.